contract_comm/random/random.go

```diff
@@ -199,3 +223,9 @@ func Random(header *types.Header, state vm.StateDB) (common.Hash, error) {
 	_, err := contract_comm.MakeStaticCall(params.RandomRegistryId, randomFuncABI, "random", []interface{}{}, &randomness, params.MaxGasForComputeCommitment, header, state)
 	return randomness, err
 }
+
+func HistoricalRandom(header *types.Header, state vm.StateDB, blockNumber uint64) (common.Hash, error) {
+	randomness := common.Hash{}
+	_, err := contract_comm.MakeStaticCall(params.RandomRegistryId, historicRandomFuncABI, "getBlockRandomness", []interface{}{big.NewInt(int64(blockNumber))}, &randomness, params.MaxGasForComputeCommitment, header, state)
```

Comment:
Is this the right amount of gas to provide?

Reply:
I added another constant called `MaxGasForBlockRandomness`, mostly just copied it from the existing implementation